infra/integration-image/Dockerfile

```diff
@@ -8,7 +8,7 @@
 # docker tag vic-test 192.168.31.15/library/vic-integration-test:1.x
 # docker push 192.168.31.15/library/vic-integration-test:1.x
 
-FROM golang:1.8
+FROM golang:1.9
```

Comment:
You need to rebuild this and upload it, it needs to be vic-integration-test:1.36 currently.  And we need this to go in after #6109 (and make sure you build the image with the changes from 6109)